packages/data-service/src/run-command.ts

```diff
@@ -223,6 +223,10 @@ export type ListCollectionsResult<CollectionType> = {
   cursor: { firstBatch: CollectionType };
 };
 
+export type listStreamProcessorsOptions = {
```

Comment:
Very tiny nit, we usually use upper camel case for types:

```suggestion
export type ListStreamProcessorsOptions = {
```

Reply:
Ah, good spot. I missed that.